### PR TITLE
Add "Who triggered build" info, add settings enable/disable this info

### PR DESCRIPTION
**Pull request by 2m0nd**

### CODE REVIEW

File: tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/payload/content/SlackNotificationPayloadContent.java [hunk 1]
```diff
@@ -329,6 +330,14 @@ public void setBuildName(String buildName) {
 			this.buildName = buildName;
 		}
 
+		public String getTriggeredBy() {
+			return triggeredBy;
+		}
+
+		public void setTriggeredBy(String triggeredBy) {
+			this.triggeredBy = triggeredBy;
+        }
```
Comment (marvin-w) [inline tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/payload/content/SlackNotificationPayloadContent.java]: Please fix the identation

File: tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/SlackNotificationListener.java [hunk 2]
```diff
@@ -89,7 +90,8 @@ public void getFromConfig(SlackNotification slackNotification, SlackNotification
             slackNotification.setShowBuildAgent(slackNotificationConfig.getContent().getShowBuildAgent());
             slackNotification.setShowElapsedBuildTime(slackNotificationConfig.getContent().getShowElapsedBuildTime());
             slackNotification.setShowCommits(slackNotificationConfig.getContent().getShowCommits());
-            slackNotification.setShowCommitters(slackNotificationConfig.getContent().getShowCommitters());
+			slackNotification.setShowCommitters(slackNotificationConfig.getContent().getShowCommitters());
+			slackNotification.setShowTriggeredBy(slackNotificationConfig.getContent().getShowTriggeredBy());
```
Comment (marvin-w) [inline tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/SlackNotificationListener.java]: Please fix the identation

File: tcslackbuildnotifier-core/src/main/java/slacknotifications/SlackNotificationImpl.java [hunk 3]
```diff
@@ -347,6 +348,10 @@ private void postViaWebHook() throws IOException {
                 attachment.addField("Changes By", committersString, false);
             }
         }
+        
+        if(showTriggeredBy){
```
Comment (marvin-w) [inline tcslackbuildnotifier-core/src/main/java/slacknotifications/SlackNotificationImpl.java]: Please add whitespaces - if (showTriggeredBy)

File: tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/settings/SlackNotificationConfig.java [hunk 4]
```diff
@@ -180,6 +181,9 @@ public SlackNotificationConfig(Element e) {
             }
             if (eContent.getAttribute(SHOW_COMMITTERS) != null){
                 this.content.setShowCommitters(Boolean.parseBoolean(eContent.getAttributeValue(SHOW_COMMITTERS)));
+			}
```
Comment (marvin-w) [inline tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/settings/SlackNotificationConfig.java]: Please fix the identation

File: tcslackbuildnotifier-web-ui/src/main/java/slacknotifications/teamcity/extension/SlackNotificationAjaxEditPageController.java [hunk 5]
```diff
@@ -175,6 +175,9 @@ protected ModelAndView doHandle(HttpServletRequest request, HttpServletResponse
                                             content.setShowCommitters((request.getParameter("showCommitters") != null)
                                                     && ("on".equalsIgnoreCase(request.getParameter("showCommitters"))));
 
+											content.setShowTriggeredBy((request.getParameter("showTriggeredBy") != null)
```
Comment (marvin-w) [inline tcslackbuildnotifier-web-ui/src/main/java/slacknotifications/teamcity/extension/SlackNotificationAjaxEditPageController.java]: Please fix the identation

File: tcslackbuildnotifier-web-ui/src/main/java/slacknotifications/teamcity/extension/bean/SlacknotificationConfigAndBuildTypeListHolder.java [hunk 6]
```diff
@@ -60,7 +61,8 @@ public SlacknotificationConfigAndBuildTypeListHolder(SlackNotificationConfig con
         showBuildAgent = valueOrFallback(config.getContent().getShowBuildAgent(), valueOrFallback(mainSettings.getShowBuildAgent(), SlackNotificationContentConfig.DEFAULT_SHOW_BUILD_AGENT));
         showElapsedBuildTime = valueOrFallback(config.getContent().getShowElapsedBuildTime(), valueOrFallback(mainSettings.getShowElapsedBuildTime(), SlackNotificationContentConfig.DEFAULT_SHOW_ELAPSED_BUILD_TIME));
         showCommits = valueOrFallback(config.getContent().getShowCommits(), valueOrFallback(mainSettings.getShowCommits(), SlackNotificationContentConfig.DEFAULT_SHOW_COMMITS));
-        showCommitters = valueOrFallback(config.getContent().getShowCommitters(), valueOrFallback(mainSettings.getShowCommitters(), SlackNotificationContentConfig.DEFAULT_SHOW_COMMITTERS));
+		showCommitters = valueOrFallback(config.getContent().getShowCommitters(), valueOrFallback(mainSettings.getShowCommitters(), SlackNotificationContentConfig.DEFAULT_SHOW_COMMITTERS));
+		showTriggeredBy = valueOrFallback(config.getContent().getShowTriggeredBy(), valueOrFallback(mainSettings.getShowTriggeredBy(), SlackNotificationContentConfig.DEFAULT_SHOW_TRIGGERED_BY));
```
Comment (marvin-w) [inline tcslackbuildnotifier-web-ui/src/main/java/slacknotifications/teamcity/extension/bean/SlacknotificationConfigAndBuildTypeListHolder.java]: Please fix the identation

**Comment by 2m0nd** [comment]
Hi! I fix all your proposals.

**Comment by marvin-w** [comment]
Thanks :).

I will make a release when I'm home.

**Pull request merged**